app/scripts/modules/core/pipeline/config/stages/bake/aws/bakeStage.html

```diff
@@ -133,7 +133,7 @@ <h4 ng-if="viewState.loading" class="text-center">
       </div>
       <div class="form-group">
         <div class="col-md-2 col-md-offset-1 sm-label-left">
-          <b>Use Enhanced Networking</b>
+          <b>Enhanced Networking</b>
           <help-field key="pipeline.config.bake.enhancedNetworking"></help-field>
         </div>
         <div class="col-md-6">
```

Comment:
I like this change, but it makes the misalignment of the checkbox a little more pronounced. As part of the PR, can you wrap the checkbox in some bootstrap classes to align it with the label?
```html
<input type="checkbox"
       ng-model="stage.enhancedNetworking"/>
```
would become
```html
<div class="checkbox">
  <label>
    <input type="checkbox"
           ng-model="stage.enhancedNetworking"/>
  </label>
</div>
```